unicode-transforms.cabal

```diff
@@ -172,11 +177,20 @@ benchmark bench
       base >=4.7 && <5
     , deepseq >=1.1.0 && <1.5
     , filepath >=1.0 && <2
-    , gauge >=0.2.0 && <0.3
     , path >=0.0.0 && <0.9
     , path-io >=0.1.0 && <1.7
     , text >=1.1.1 && <1.3
     , unicode-transforms
+  if flag(use-gauge)
+    build-depends: gauge >=0.2.0 && <0.3
+  else
+    build-depends: tasty-bench>= 0.2.5 && <0.4
+
+                   -- unicode-transforms does not depend on process itself but
+                   -- some of its dependincies do. process-1.6.13.0 fails to
+                   -- build and this is basically a hack for the time being.
+                 , process < 1.6.13.0 && > 1.6.13.0
```

Comment:
Did you try `cabal update`? `process-1.6.13.0` is already deprecated and should not be chosen by `cabal` any longer.

Reply:
Force pushed reverting the change. The CI script does do a `cabal update` I believe.